### PR TITLE
Add sounds when player drops flag

### PR DESCRIPTION
**Pull request by a-blob**
I have no idea what I'm doing.

### CODE REVIEW

**Comment by a-blob** [comment]
Still have to change the sounds when the player drops the flag. The code is using the old trumpet sound for now as a test.

**Comment by farooqkz** [comment]
The sounds look good to me but I'm not a professional anyway.

File: mods/ctf/ctf_modebase/features.lua [hunk 1]
```diff
@@ -116,6 +116,27 @@ local function celebrate_team(teamname)
...
+				gain = 1.0,
+				pitch = 1.0,
+			}, true)
+		else
+			minetest.sound_play("ctf_modebase_drop_flag_positive", {
+				to_player = pname,
+				gain = 1.0,
+				pitch = 1.0,
+			}, true)
+		end
```
Comment (LoneWolfHT) [inline mods/ctf/ctf_modebase/features.lua:136]:
```suggestion
        if pteam then
		  if pteam == teamname then
			  minetest.sound_play("ctf_modebase_drop_flag_negative", {
				  to_player = pname,
				  gain = 1.0,
				  pitch = 1.0,
			  }, true)
		  else
			  minetest.sound_play("ctf_modebase_drop_flag_positive", {
				  to_player = pname,
				  gain = 1.0,
				  pitch = 1.0,
			  }, true)
		  end
		end
```

**Comment by LoneWolfHT** [comment]
Changing it to use the default sounds for now, with gain/pitch shifting. It sounds ok, but feel free to propose alternatives

**Pull request merged**